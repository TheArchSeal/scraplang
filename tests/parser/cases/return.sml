return;
return 0;
break;
continue;
