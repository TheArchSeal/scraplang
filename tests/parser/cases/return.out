stmt (14):1:1 return
    expr (1):1:1 (empty)
stmt (14):2:1 return
    expr (3):2:8 0
stmt (15):3:1 break
stmt (16):4:1 continue
